_posts/2019-07-22-aws-event-bridge.md

```diff
@@ -0,0 +1,69 @@
+---
+layout: post
+title: AWS Event Bridge Introduced
+date: 2019-03-29
+tags: aws event-bridge
+author: JK Gunnink
+---
+
+Earlier this month Amazon Web Services introduced a new service to their suite called EventBridge,
+and amongst the community it's created quite a buzz. [Some are even claiming it to be the biggest
+thing since Lambda](https://www.trek10.com/blog/amazon-eventbridge/).
+
+## What is it?
+
+According to the EventBridge landing page:
+
+> Amazon EventBridge is a serverless event bus that makes it easy to connect applications together
+> using data from your own applications, Software-as-a-Service (SaaS) applications, and AWS
+> services. EventBridge delivers a stream of real-time data from event sources, such as Zendesk,
+> Datadog, or Pagerduty, and routes that data to targets like AWS Lambda. You can set up routing
+> rules to determine where to send your data to build application architectures that react in real
+> time to all of your data sources. EventBridge makes it easy to build event-driven applications
+> because it takes care of event ingestion and delivery, security, authorization, and error handling
+> for you.
+
+Building on the CloudWatch service which can be used a an event trigger, EventBridge allows events
+from AWS and external services to be captured by EventBridge and forwarded on to a destination that
+fits the event's use case based on it's contents.
+
+Some architectures and patterns come to mind when dealing with events, one such example is the
+rapids, rivers and ponds architecture. EventBridge allows you to filter events from many sources, at
```

Comment:
"rapids, rivers and ponds architecture" - Link to an article or blog post.  Fred George has his YOW nights talk - where I originally saw it.  But should link to one of our posts where possible - a post may not currently exist though.

Reply:
Can I link to your Latency talk @temyers ? 😄